src/status_im/ui/screens/contacts/events.cljs

```diff
@@ -385,8 +381,4 @@
   (fn [{:keys [db]}]
     (let [{:contacts/keys [new-identity]} db]
       (when (and new-identity (not (string/blank? new-identity)))
-        {:dispatch (if (get-in db [:contacts/contacts new-identity])
-                     [:add-pending-contact-and-open-chat new-identity]
-                     [:add-new-contact-and-open-chat {:name             (generate-gfy new-identity)
-                                                      :photo-path       (identicon new-identity)
-                                                      :whisper-identity new-identity}])}))))
+        {:dispatch [:add-contact-and-open-chat new-identity]}))))
```

Comment:
maybe better to have `add-contact-and-open-chat` function and do not dispatch extra event